exams/pearson/constants.py

```diff
@@ -6,6 +6,23 @@
 
 PEARSON_FILE_TYPE_EAC = 'eac'
 PEARSON_FILE_TYPE_VCDC = 'vcdc'
+PEARSON_FILE_TYPE_EXAM = 'exam'
+PEARSON_FILE_TYPE_CAND = 'cand'
+PEARSON_FILE_TYPE_SURV = 'surv'
+PEARSON_FILE_TYPE_CMNT = 'cmnt'
+PEARSON_FILE_TYPE_SECT = 'sect'
+PEARSON_FILE_TYPE_RESP = 'resp'
+PEARSON_FILE_TYPE_ITEM = 'item'
+
+# these are files we intentionally skip, but treat as if we processed them
+PEARSON_INTENDED_SKIP_FILE_TYPES = (
+    PEARSON_FILE_TYPE_CAND,
+    PEARSON_FILE_TYPE_SURV,
+    PEARSON_FILE_TYPE_CMNT,
+    PEARSON_FILE_TYPE_SECT,
+    PEARSON_FILE_TYPE_ITEM,
+    PEARSON_FILE_TYPE_RESP,
+)
```

Comment:
Not `namedtuple`, sorry (although it _could_ be used). I meant `SimpleNamespace` ([doc](https://docs.python.org/3/library/types.html#types.SimpleNamespace)). It would look something like this:

```python
PEARSON_FILE_TYPES = SimpleNamespace(
    EAC='eac',
    VCDC='vcdc',
    # ...
)

# Usage:
from exams.pearson.constants import PEARSON_FILE_TYPES
def is_eac_file_type(file_type):
    if filetype == PEARSON_FILE_TYPES.EAC:
        return True
```